Weater-app/src/components/customButton/styledComponents/CustomButton.js

```diff
@@ -1,9 +1,34 @@
 import styled from 'styled-components';
 
-import { DarkGray } from '../../../assets/colors/colors';
+import {
+  Black,
+  DarkGray,
+  DodgerBlue,
+  LightBlack,
+  White,
+} from '../../../assets/colors/colors';
 
 export const BtnWrapper = styled.div`
   display: flex;
+  align-items: center;
+
+  .btn {
+    font-size: 22px;
+    padding: 7px;
+    cursor: pointer;
+    border: none;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+    background: ${(props) =>
+      props.changeTheme ? `${LightBlack}` : `${White}`};
```

Comment:
changeTheme - bad name again: it just says we need to change theme, but what if we have 3,4,5 themes instead of 2? better to use enums or similar

but in general - it's bad idea to use `if`s for theme because it's a lot of duplications, decrease readability and we need to keep always in mind that we need to support it. take a look at themeprovider in material ui. maybe we can use it or make similar thing in this project